charts/rasa-x/values.yaml

```
@@ -191,6 +191,19 @@ rasa:
   versions:
     # rasaProduction is the container which serves the production environment
     rasaProduction:
+
+      # enable rasa-production deployment
+      # You can disable the rasa-production deployment in order to use external Rasa OSS deployment instead.
+      enabled: true
+
+      # Define if external Rasa OSS should be used.
+      external:
+        # enable external Rasa OSS
+        enabled: true
+
+        # host of external Rasa OSS deployment
+        host: "http://rasa-bot"
+
```

```suggestion
      # You can disable the rasa-production deployment in order to use external Rasa OSS deployment instead.
      enabled: true

      # Define if external Rasa OSS should be used.
      external:
        # enable external Rasa OSS
        enabled: false

        # host of external Rasa OSS deployment
        host: "http://rasa-bot"

```
default should be false if enabled is default true